frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1582,7 +1579,7 @@ With this enabled, the current page is included, so the count goes from n to 1 i
                 local battery_threshold = SpinWidget:new{
                     value = self.settings.battery_hide_threshold,
                     value_min = 0,
-                    value_max = 100,
+                    value_max = Device:hasAuxBattery() and 200 or 100,
```

Comment:
Does hasAuxBattery() means it can have a aux battery ? Or that it actually have a aux battery plugged?
No issue with a user setting this threshold to 140 while a aux battery was plugged, and later unplugging it and living without it ?

Reply:
Only that it can.

Since this a somewhat long-lived setting, I don't really think making it take into account the current state of the PowerCover would make sense.

(But, granted, the whole thing is a bit annoying: I had to scratch my head for a while to figure out why I wasn't seeing the widget, since this defaults to 100 ;p).